src/components/PaginatedTable/index.tsx

```diff
@@ -13,6 +14,8 @@ export interface Props {
   itemsOnCurrentPage: React.ReactNode[][];
   totalNumberOfItems: number;
   variablesClassName?: string;
+  buttonVariablesClassName?: string;
```

Comment:
hey @branquinhoaa upon closer reflection, I've noticed that we don't need a `buttonVariablesClassName` or a `paginationVariableClassName`. We can simply replace them with the regular `variablesClassName`. The reason for this is because we're expecting the changes to through the variables we're passing to the component. Because of how we've set up our CSS, even if a button variable is being called in another location, it won't take effect through a side-effect.